x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -83,6 +85,11 @@ function useServicesMainStatisticsFetcher(searchQuery: string | undefined) {
               searchQuery,
             },
           },
+        }).then((mainStatisticsData) => {
+          return {
+            requestId: uuidv4(),
+            ...mainStatisticsData,
+          };
```

Comment:
Why did you re-add this? `requestId` shouldn't be necessary - all we need to look at is whether the items changed, then re-fetch if they did.

Reply:
Just commenting on what we talked about and why we agreed that the `requestId` is necessary:

The requestId makes necessary because it invalidates the **detailed statistics fetcher** in case the user clicks on the Refresh button and the same number of transactions are returned.